suites/reef/rados/tier-2_rados_test-stretch-mode.yaml

```diff
@@ -61,41 +121,28 @@ tests:
               args:
                 placement:
                   label: rgw
-          - config:
-              command: apply
-              service: mds
-              base_cmd_args:
-                verbose: true
-              pos_args:
-                - cephfs
-              args:
-                placement:
-                  nodes:
-                    - node2
-                    - node6
-                  limit: 2
-                  sep: " "
-      destroy-cluster: false
-      abort-on-fail: true
 
   - test:
       name: Configure client admin
       desc: Configures client admin node on cluster
       module: test_client.py
-      polarion-id: CEPH-83573758
+      polarion-id:
       config:
         command: add
-        id: client.1
-        node: node7
+        id: client.1                      # client Id (<type>.<Id>)
+        nodes:
+          - node8:
+              release: 6
```

Comment:
If this is a mistake, please change release to 7

Reply:
Updated